core/src/main/kotlin/no/ntnu/beardblaster/view/RegisterScreen.kt

```diff
@@ -18,20 +24,106 @@ class RegisterScreen(game: BeardBlasterGame) : AbstractScreen(game) {
         Gdx.input.inputProcessor = result
         result
     }
+
+
+    private lateinit var skin: Skin
+    private lateinit var table: Table
+    private lateinit var heading: Label
+
+    private lateinit var checkButton: TextButton
+    private lateinit var unCheckButton: TextButton
+    private lateinit var createButton: TextButton
+
+    private lateinit var userNameInput: TextField
+    private lateinit var emailInput: TextField
+    private lateinit var passwordIput: TextField
+    private lateinit var rePasswordIput: TextField
     override fun show() {
         LOG.debug { "Registration Screen Shown" }
+
+        skin = Skin(Assets.assetManager.get(Assets.atlas))
+        table = Table(skin)
+        table.setBounds(0f,0f, viewport.worldWidth, viewport.worldHeight)
+
+
+        var standardFont = Assets.assetManager.get(Assets.standardFont)
+
+
+        val headingStyle = Label.LabelStyle(standardFont, Color.BLACK).also {
+            heading = Label("Create Wizard", it)
+            heading.setFontScale(2f)
+            it.background = skin.getDrawable("modal_fancy_header")
+            heading.setAlignment(Align.center)
+        }
+
+        val checkButtonStyle = TextButton.TextButtonStyle()
+        val unCheckButtonStyle = TextButton.TextButtonStyle()
+        val createUserButtonStyle = TextButton.TextButtonStyle()
+        skin.getDrawable("modal_fancy_header_button_green").also { checkButtonStyle.up = it }
+        skin.getDrawable("modal_fancy_header_button_green_check").also { checkButtonStyle.down = it }
+        skin.getDrawable("modal_fancy_header_button_red").also { unCheckButtonStyle.up = it }
+        skin.getDrawable("modal_fancy_header_button_red_cross").also { unCheckButtonStyle.down = it }
+        skin.getDrawable("button_fancy_dark_short").also { createUserButtonStyle.down = it }
+        skin.getDrawable("button_fancy_dark_short").also { createUserButtonStyle.up = it }
+
+        standardFont.apply {
+            checkButtonStyle.font = this
+            unCheckButtonStyle.font = this
+            createUserButtonStyle.font = this
+        }
+
+        checkButton = TextButton("", checkButtonStyle)
+        unCheckButton = TextButton("", unCheckButtonStyle)
+        createButton = TextButton("Create Wizard", createUserButtonStyle)
+
+        val textInputStyle = TextField.TextFieldStyle()
+
+        textInputStyle.also {
+            it.background = skin.getDrawable("input_texture_dark")
```

Comment:
Mulig input_texture_light ser bedre ut? 💅 Dunno!

```suggestion
            it.background = skin.getDrawable("input_texture_light")
```